algebird-test/src/test/scala/com/twitter/algebird/macros/CuberMacroTest.scala

```diff
@@ -0,0 +1,67 @@
+package com.twitter.algebird.macros
+
+import com.twitter.algebird._
+import com.twitter.algebird.macros._
+import com.twitter.algebird.macros.ArbitraryCaseClassMacro.arbitrary
+
+import org.scalacheck.{ Properties, Arbitrary }
+import org.scalacheck.Prop.forAll
+
+object CuberRollerProperties extends Properties("Cuber and roller macro") {
```

Comment:
He's not a huge fan of scalatest its true, but most of our tests are already scalatest, and it provides metrics on how many run. A big pain point is when tests don't get run, so it would provide us that data so we can graph/track in future.